doc/classes/AudioStreamPlayback.xml

```diff
@@ -79,5 +79,46 @@
 				Overridable method. Called whenever the audio stream is mixed if the playback is active and [method AudioServer.set_enable_tagging_used_audio_streams] has been set to [code]true[/code]. Editor plugins may use this method to "tag" the current position along the audio stream and display it in a preview.
 			</description>
 		</method>
+		<method name="get_loop_count" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the number of times the stream has looped.
+			</description>
+		</method>
+		<method name="get_playback_position" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the current position in the stream, in seconds.
+			</description>
+		</method>
+		<method name="get_raw_audio">
```

Comment:
I would just call it mix_audio()

Reply:
Changed, thanks

Reply:
@stechyo is this PR still pending the above change?

Reply:
No, the change was made when I wrote the reply above (June 16).